test/src/test/java/org/corfudb/integration/LogReplicationAbstractIT.java

```diff
@@ -161,11 +166,11 @@ public void testEndToEndSnapshotAndLogEntrySync() throws Exception {
 
     }
 
-    public void testEndToEndSnapshotAndLogEntrySyncUFO(boolean diskBased) throws Exception {
-        testEndToEndSnapshotAndLogEntrySyncUFO(1, diskBased);
+    public void testEndToEndSnapshotAndLogEntrySyncUFO(boolean diskBased, boolean checkRemainingOnDeltaSnapshotSync) throws Exception {
```

Comment:
this name is misleading, check on delta or on snapshot? or both? also let's try to keep it as Log Entry (delta).  

Also, the real meaning of this variable seems to be retriggerSnapshotSync, so indirectly you can re-compute remaining entries from the second log entry sync cycle, is this right?

Reply:
yeah, I am not happy with the name as well. Used "delta" because the name is already pretty big! I'll give some more thought and use log_entry itself.

And yes, that is the meaning.